From: You-Sheng Yang <vicamo@gmail.com>
Date: Mon, 15 Jul 2019 17:56:56 +0800
Subject: fix format-overflow compile error in kconf/confdata.c

---
 kconf/confdata.c | 7 +++++--
 1 file changed, 5 insertions(+), 2 deletions(-)

diff --git a/kconf/confdata.c b/kconf/confdata.c
index 297b079..ed1ff5d 100644
--- a/kconf/confdata.c
+++ b/kconf/confdata.c
@@ -770,10 +770,13 @@ int conf_write(const char *name)
 	} else
 		basename = conf_get_configname();
 
-	sprintf(newname, "%s%s", dirname, basename);
+	if (snprintf(newname, sizeof newname, "%s%s", dirname, basename) < 0)
+		return 1;
+
 	env = getenv("KCONFIG_OVERWRITECONFIG");
 	if (!env || !*env) {
-		sprintf(tmpname, "%s.tmpconfig.%d", dirname, (int)getpid());
+		if (snprintf(tmpname, sizeof tmpname, "%s.tmpconfig.%d", dirname, (int)getpid()) < 0)
+			return 1;
 		out = fopen(tmpname, "w");
 	} else {
 		*tmpname = 0;
